third_party/closure-library/compile.sh

```diff
@@ -0,0 +1,37 @@
+# Run this file with bash to generate new closure based compiled
+# fragments for inclusion into AMP.
+
+# $1 is a path to a closure compiler jar file.
+# $2 is a path to a closure library directory.
+
+
+readonly GIT_COMMIT=`sh -c "cd $2 && git log --pretty=format:'%H' -n 1"`
+
+
+# We manually list deps, because this yields a 1KB smaller file size
+# over automatically managed dependencies.
+# As an output wrapper we add and export at the end, so we can use this
+# file from ES6 modules.
+java -jar $1 \
```

Comment:
I kinda think it is better to keep it here, but could be convinced otherwise.

We definitely want to revisit this approach when we actually compile with closure compiler. This is only temporary.